src/lib.js

```diff
@@ -677,6 +676,36 @@ var program,
       }
     },
 
+    /**
+     *
```

Comment:
Add what brief infor about what function does here.

For example: https://github.com/postmanlabs/curl-to-postman/blob/b7971d824ffbc6036ea102221a318ce6f3d9aac5/src/lib.js#L494